sphinxcontrib/spelling/tests/test_builder.py

```diff
@@ -129,3 +129,20 @@ def test_several_word_lists(sphinx_project):
     assert '(txt)' not in output_text
     # But not this one
     assert '(tihs)' in output_text
+
+
+def test_ignore_file(sphinx_project):
+    srcdir, outdir = sphinx_project
+    add_file(srcdir, 'conf.py', '''
+    extensions = ['sphinxcontrib.spelling']
+    spelling_ignore_filename=['contents']
+    ''')
+
+    add_file(srcdir, 'contents.rst', '''
+    Welcome to Speeling Checker documentation!
+    ==========================================
+    ''')
+
+    stdout, stderr, output_text = get_sphinx_output(srcdir, outdir, 'contents')
+    # This should be fine now
+    assert '(Speeling)' not in output_text
```

Comment:
A recent change means that `get_sphinx_output()` will return `None` for `output_text` in this case.

Reply:
Thanks, this should be fixed now.